tests/Wallabag/CoreBundle/WallabagCoreTestCase.php

```diff
@@ -65,23 +75,42 @@ public function logInAsUsingHttp($username)
     }
 
     /**
-     * Return the user id of the logged in user.
+     * Return the user of the logged in user.
      * You should be sure that you called `logInAs` before.
      *
-     * @return int
+     * @return User
      */
-    public function getLoggedInUserId()
+    public function getLoggedInUser()
     {
         $token = static::$kernel->getContainer()->get('security.token_storage')->getToken();
 
         if (null !== $token) {
-            return $token->getUser()->getId();
+            return $token->getUser();
         }
 
         throw new \RuntimeException('No logged in User.');
     }
 
     /**
+     * Return the user id of the logged in user.
+     * You should be sure that you called `logInAs` before.
+     *
+     * @return int
+     */
+    public function getLoggedInUserId()
+    {
+        return $this->getLoggedInUser()->getId();
+    }
+
+    public function useTheme($theme)
+    {
+        $config = $this->getEntityManager()->getRepository(Config::class)->findOneByUser($this->getLoggedInUser());
+        $config->setTheme('baggy');
```

Comment:
`$theme` instead of `baggy` I guess.
This method might help us with https://github.com/wallabag/wallabag/issues/1373
Good idea.